components/bl-knob-component/src/index.js

```diff
@@ -18,6 +18,9 @@ export default function KnobComponent({ component, eventHandlers }) {
   const [knobDisabled, setKnobDisabled] = useState(!!disabled);
 
   useEffect(() => {
```

Comment:
Are you sure this is working?

For me it's looks not valid.

Reply:
We need it for this case 
![image](https://user-images.githubusercontent.com/29707734/216386813-16d10fe3-9052-46d8-b683-a8096e442ed4.png)

Reply:
Yea, I understand why we need this.

Look please at code - it's not valid, just try to run it.

```
  useEffect(() => {
    setKnobReadOnly(!!readOnly);
  },[readOnly]);

    setKnobDisabled(!!readOnly);
  }, [disabled]);
```